pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -237,30 +237,17 @@ func (p *ProjectService) volumes(project *composego.Project) ([]Volumes, error)
 	}
 
 	for i, vol := range vols {
-		size, selector, storageClass := getVolumeLabels(project.Volumes[vol.VolumeName])
+		composeVol := project.Volumes[vol.VolumeName]
+		k8sVol, err := config.VolK8sConfigFromCompose(&composeVol)
+		if err != nil {
+			return nil, err
+		}
 
 		// We can't assign value to struct field in map while iterating over it, so temporary variable `temp` is used here
 		var temp = vols[i]
-
-		// set PVC size from label if present, or default size
-		if len(size) > 0 {
-			temp.PVCSize = size
-		} else {
-			temp.PVCSize = config.DefaultVolumeSize
-		}
-
-		// set PVC selector from label if present
-		if len(selector) > 0 {
-			temp.SelectorValue = selector
-		}
-
-		// set PVC storage class from label if present, or default class
-		if len(storageClass) > 0 {
```

Comment:
with default storage class unspecified (empty string) I guess we'd want to keep this conditional?

Reply:
Not really, the defaults are now set in `config.VolK8sConfigFromCompose(&composeVol)`.

So if the default is needed, as in storage class didn't start with a value, it will be set there.